JSON/README.md

```diff
@@ -13,45 +13,42 @@ JSON schemas for all the config files and objects used by Tyk products.
 
 ## Client to Gateway Authentication
 
-| Type        | JSON      | YAML | Comments | Draft |
-| ----------- | --------- | ---- | --------- | -------- |
-| Tyk API definition | ⚠️ | ❌️ | - | 07 |
-| Tyk API OAS definition | ❌️ | ❌️ | - | 07 |
-| Tyk API definition in Tyk operator | ❌️ | ❌ | - | 07 |
-| Tyk key definition | ⚠️ | ❌️ | Also referred as "session object" | 07 |
-| Tyk policy definition | ❌️ | ❌️ | - | 07 |
-| Tyk OSS gateway config file | ⚠️ | NA | - | 07 |
-| Tyk Pro gateway config file | ❌️ | NA | - | 07 |
-| Tyk hybrid gateway config file | ❌️ | NA | - | 07 |
-| Tyk pump config file | ❌️ | NA | - | 07 |
-| Tyk dashboard config file | ❌️ | NA | - | 07 |
-| Tyk Identity broker config file | ❌️ | NA | - | 07 |
+| Schema Description        | JSON      | YAML | Comments | Draft | When is it used
+| ----------- | --------- | ---- | --------- | -------- | -------- |
+| Tyk Gateway API definition | ⚠️ | ❌️ | - | 07 | An OpenAPI Spec object |
+| Tyk dashboard API definition | ⚠️ | ❌️ | - | 07 | An OpenAPI Spec object |
+| Tyk API OAS definition | ❌️ | ❌️ | - | 07 | An OpenAPI Spec object |
+| Tyk dashboard API OAS definition | ⚠️ | ❌️ | - | 07 | An OpenAPI Spec object |
+| Tyk API definition in Tyk operator | ❌️ | ❌ | - | 07 | Tyk operator object |
+| Tyk Gateway key definition | ⚠️ | ❌️ | Also referred as "session object" | 07 | An OpenAPI Spec object |
+| Tyk dashboard key definition | ⚠️ | ❌️ | Also referred as "session object" | 07 | An OpenAPI Spec object |
+| Tyk Gateway policy definition | ❌️ | ❌️ | - | 07 | An OpenAPI Spec object |
+| Tyk dashboard policy definition | ❌️ | ❌️ | - | 07 | An OpenAPI Spec object |
+| Tyk OSS gateway config file | ⚠️ | NA | Examples and default values specific to gateway in an open source deployment | 07 | Config file |
+| Tyk Self-managed gateway config file | ❌️ | NA | Examples and default values specific to gateway in a self managed deployment | 07 | Config file |
+| Tyk hybrid gateway config file | ❌️ | NA | Examples and default values specific to gateway in a hybrid deployment | 07 | Config file |
+| Tyk pump config file | ❌️ | NA | - | 07 | Config file |
+| Tyk dashboard config file | ❌️ | NA | - | 07 | Config file |
+| Tyk Identity broker config file | ❌️ | NA | - | 07 | Config file |
+| Tyk Self-managed user | ❌️ | NA | - | 07 | An OpenAPI Spec object |
+| Tyk Self-managed user group | ❌️ | NA | - | 07 | An OpenAPI Spec object |
 
 # Example for usefull usage
-To write in "Tyk language" and feel it's native to your IDE configure the settings in your IDE to use the JSON schemas in this repository
 
-## VSCode
+1. Write correct and valid API calls.
+2. Write code that etend Tyk and make sure the input objects are correct.
```

Comment:
I am unsure, but did you mean `extend` instead of `etend`? If not, please ignore this 
```suggestion
2. Write code that extends Tyk and make sure the input objects are correct.
```